Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -491,18 +491,15 @@ static void CheckForInactiveConnections()
 
         static void AddTransportHandlers()
         {
-            Transport.activeTransport.OnServerDisconnected.AddListener(OnDisconnected);
-            Transport.activeTransport.OnServerConnected.AddListener(OnConnected);
-            Transport.activeTransport.OnServerDataReceived.AddListener(OnDataReceived);
-            Transport.activeTransport.OnServerError.AddListener(OnError);
+            Transport.activeTransport.OnServerConnected = OnConnected;
+            Transport.activeTransport.OnServerDataReceived = OnDataReceived;
+            Transport.activeTransport.OnServerDisconnected = OnDisconnected;
+            Transport.activeTransport.OnServerError = OnError;
         }
 
         static void RemoveTransportHandlers()
```

Comment:
this function can safely be removed in this PR.
it existed because we previously called AddListener and so starting networkclient twice would add listeners twice.
now we do event = callback, which means we only ever add it once.
so the Transport.Action change makes NetworkClient a lot easier too :)

Reply:
If we dont clear/reset them on stop then we would need extra code somewhere else to make sure that the handlers in NetworkClient/NetworkServer are not called after stop is called

Reply:
that's fine, nobody will call them after stopping. that's the point of stopping :)